src/DurableTask.Netherite/StorageLayer/Faster/AzureBlobs/BlobManager.cs

```diff
@@ -187,7 +218,7 @@ class StorageFormatSettings
             Formatting = Formatting.None,
         };
 
-        public static void CheckStorageFormat(string format, NetheriteOrchestrationServiceSettings settings)
+        public static void CheckAndLoadStorageFormat(string format, NetheriteOrchestrationServiceSettings settings)
```

Comment:
super nitpick: at first glance, I find it a bit unintuitive to first _check_ and then _load_ the config here. The opposite order makes more sense to me, as it allows us to validate the config we just loaded. I realize this current ordering works because we don't really throw any validation errors for the config we _load_.

I'll make a _soft_ suggestion to rename this to `LoadAndCheckStorageFormat` and check the order of operations to match that new name, which I think is more future-proof. But feel free to disregard this comment as it's mostly a stylistic nit in the present.

Reply:
I have reworked this a bit to make it clearer.